sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents.py

```diff
@@ -43,12 +44,16 @@ def analyze_custom_documents(custom_model_id):
     key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
     model_id = os.getenv("CUSTOM_BUILT_MODEL_ID", custom_model_id)
 
-    document_intelligence_client = DocumentIntelligenceClient(endpoint=endpoint, credential=AzureKeyCredential(key))
+    document_intelligence_client = DocumentIntelligenceClient(
+        endpoint=endpoint, credential=AzureKeyCredential(key)
+    )
```

Comment:
Pls avoid formatting the previous code. Only format yours.

Reply:
ok, will notice that